src/preferences.md

```diff
@@ -50,18 +50,19 @@ deck.
 
 ## Scheduling
 
-**Show learning cards with larger steps before**  
-Only shown when the 2.1 scheduler is enabled. Normally learning cards with a 1+
-day delay are shown after normal reviews. If checked, Anki will show them before
-normal reviews instead.
-
-**Anki 2.1 scheduler**  
-The Anki 2.1 scheduler is documented here:
-<https://faqs.ankiweb.net/the-anki-2.1-scheduler.html>
-
-**Mix new cards and reviews**:
-This drop-down box controls when new cards are shown: either mixed
-with, before, or after all reviews.
```

Comment:
This setting will still be shown when the v1/v2 schedulers are enabled, so it's too early to remove this. Same for "show learning cards with larger steps" (v2)

Reply:
Done, thanks!